buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -0,0 +1,112 @@
+    blendmode: int
+    source_rect: Rect
+    visible: int
+    layer: int
+    def _set_visible(self, value: int) -> None: ...
+    def _get_visible(self) -> int: ...
+
+class AbstractGroup:
+    spritedict = Dict[Sprite, int]
+    lostsprites = List[int]  # I think
+    def __init__(self) -> None: ...
+    def copy(self) -> AbstractGroup: ...
+    def sprites(self) -> List[Sprite]: ...
+    def add(self, *sprites: Sprite) -> None: ...
+    def remove(self, *sprites: Sprite) -> None: ...
+    def has(self, *sprites: Sprite) -> bool: ...
+    def update(self, *args, **kwargs) -> None: ...
+    def draw(self, surface: Surface) -> None: ...
+    def clear(self, surface_dest: Surface, background: Surface) -> None: ...
+    def empty(self) -> None: ...
+
+class Group(AbstractGroup):
+    def __init__(self, *sprites: Sprite) -> None:
+        AbstractGroup.__init__(self)
+    def copy(self) -> Group: ...
+class RenderPlain(Group):
+    def copy(self) -> RenderPlain: ...
+class RenderClear(Group):
+    def copy(self) -> RenderClear: ...
+class RenderUpdates(Group):
+    def copy(self) -> RenderUpdates: ...
+    def draw(self, surface: Surface) -> List[Rect]: ...
+class OrderedUpdates(RenderUpdates):
+    def copy(self) -> OrderedUpdates: ...
+
+class LayeredUpdates(AbstractGroup):
+    def __init__(self, *sprites: Sprite, **kwargs: Dict[str, Any]) -> None:
+        AbstractGroup.__init__(self)
+    def copy(self) -> LayeredUpdates: ...
+    def add(self, *sprites: Sprite, **kwargs: Dict[str, Any]) -> None: ...
+    def draw(self, surface: Surface) -> List[Rect]: ...
+    def get_sprites_at(self, pos: Union[Tuple[int, int], List[int]]) -> List[Sprite]: ...
+    def get_sprite(self, idx: int) -> Sprite: ...
+    def remove_sprites_of_layer(self, layer_nr: int) -> List[Sprite]: ...
+    def layers(self) -> List[int]: ...
+    def change_layer(self, sprite: Sprite, new_layer: int) -> None: ...
+    def get_layer_of_sprite(self, sprite: Sprite) -> int: ...
+    def get_top_layer(self) -> int: ...
+    def get_bottom_layer(self) -> int: ...
+    def move_to_front(self, sprite: Sprite) -> None: ...
+    def move_to_back(self, sprite: Sprite) ->None: ...
+    def get_top_sprite(self) -> Sprite: ...
+    def get_sprites_from_layer(self, layer: int) -> List[Sprite]: ...
+    def switch_layer(self, layer1_nr, layer2_nr) -> None: ...
+
+class LayeredDirty(LayeredUpdates):
+    def __init__(self, *sprites: DirtySprite, **kwargs: Dict[str, Any]):
+        LayeredUpdates.__init__(self, *sprites, **kwargs)
+    def copy(self) -> LayeredDirty: ...
+    def draw(self, surface: Surface, bgd: Optional[Surface]=None) -> List[Rect]: ...
+    def clear(self, surface: Surface, bgd: Surface) -> None: ...
+    def repaint_rect(self, screen_rect: _RectStyle) -> None: ...
+    def set_clip(self, screen_rect: Optional[_RectStyle]=None): ...
+    def get_clip(self) -> Rect: ...
+    def set_timing_treshold(self, time_ms: Any) -> None: ...  # This actually accept any value
```

Comment:
You fixed this in https://github.com/pygame/pygame/pull/1688

Reply:
Tnx.